Discovering Subdomains/program.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
+# the domain to scan for subdomains
+domain = "email.com" // Here we can take any domain 
+# read all subdomains
+file = open("subdomains.txt")
```

Comment:
- Open files using with
  - Opening a file using with statement is preferred as function open implements the context manager protocol that releases the resource when it is outside of the with block. Not doing so requires you to manually release the resource.

Reply:
> * Open files using with
>   
>   * Opening a file using with statement is preferred as function open implements the context manager protocol that releases the resource when it is outside of the with block. Not doing so requires you to manually release the resource.


with FileManager('test.txt', 'w') as f:    
 f.write('Test')   
print(f.closed)
--
You say this type but sir the current program is running verey smooth but on addition of something they don't work so plss help me